tests/subsys/nrf_profiler/src/main.c

```diff
@@ -101,30 +104,20 @@ static void test_performance1(void)
 	       PROFILED_EVENTS_NB, elapsed_time_us);
 }
 
-static void test_performance2(void)
+ZTEST(suite_profiler, test_performance_02)
 {
 	uint32_t elapsed_time_us = test_performance_core(profile_data_event, data_event_id);
 
 	printk("Logged %d events with 4-byte data.\nElapsed time [us]: %d\n",
 	       PROFILED_EVENTS_NB, elapsed_time_us);
 }
 
-static void test_performance3(void)
+ZTEST(suite_profiler, test_performance_03)
 {
 	uint32_t elapsed_time_us = test_performance_core(profile_big_event, big_event_id);
 
-	printk("Logged %d events with 14-byte data and 14-character string.\nElapsed time [us]: %d\n",
-	       PROFILED_EVENTS_NB, elapsed_time_us);
+	printk("Logged %d events with 14-byte data and 14-character string.\n"
+	       "Elapsed time [us]: %d\n", PROFILED_EVENTS_NB, elapsed_time_us);
 }
 
-void test_main(void)
-{
-	ztest_test_suite(nrf_profiler_tests,
-			 ztest_unit_test(test_init),
-			 ztest_unit_test(test_performance1),
-			 ztest_unit_test(test_performance2),
-			 ztest_unit_test(test_performance3)
-			 );
-
-	ztest_run_test_suite(nrf_profiler_tests);
-}
+ZTEST_SUITE(suite_profiler, NULL, test_init, NULL, NULL, NULL);
```

Comment:
you seem to change the of the test suite from nrf_profiler_tests to suite_profiler. Is it intentional?

Reply:
Yes, I wanted to name suites as `suite_xxx` and tests as `test_xxx` to use consistent naming in our tests. Changed to `suite_nrf_profiler` (to keep whole subsystem name).